app/test/unit/find-forked-remotes-to-prune-test.ts

```diff
@@ -96,8 +98,11 @@ describe('findForkedRemotesToPrune', () => {
 
     const remotesToPrune = findForkedRemotesToPrune(remotes, [], allBranches)
 
-    expect(getNamesFromRemotes(remotesToPrune)).not.toContain(
-      GitHubDesktopRemoteWithLocalBranch
+    assert.equal(
+      getNamesFromRemotes(remotesToPrune).includes(
+        GitHubDesktopRemoteWithLocalBranch
+      ),
+      false
```

Comment:
Maybe something like this?
```suggestion
    assert(
      !getNamesFromRemotes(remotesToPrune).includes(
        GitHubDesktopRemoteWithLocalBranch
      )
```